### ISSUE TITLE
ceffe2入门教程

### ISSUE THREAD

**Issue by sunnyl95** [sunnyl95/R-FCN#3]
1.安装编译完caffe后，其主目录下有：

2.训练模型之前需要先准备好训练数据MNIST，执行以下命令可以下载MNIST数据库：

3.由于caffe支持的数据类型不包括图像类型，所以常规做法需要将图像类型转为lmdb类型。
4.准备好数据之后，我们需要定义我们的网络模型，在caffe中是通过.prototxt配置文件来定义的，执行以下命令：

#caffe是如何定义网络的？

#输入层（数据层）
layer{
	name:"mnist"   //表示层名
	type："Date"   //表示层的类型
	top："data"	   
	top："lable"
	include{
		phase：TRAIN   //表示仅在训练阶段起作用
	}
	transform_param{
		scale: 0.00390625   //将图像像素归一化
	}
	data_param{
		source:"example/mnist/mnist_train_imdb"      //数据来源
		batch_size: 64								 //训练师每个迭代的输入样本数量
		backend: LMDB                                 //数据类型
	}
}

#卷积层
layer{
	name: "conv1"
	type: "Convolution"
	bottom: "data"					//输入是data
	top："conv1"					//输出是卷积特征
	param{
		lr_mult: 1					//权重参数w的学习率倍数
	}
	param{
		lr_mult: 2					//偏执参数b的学习率倍数
	}
	convolution_param{
		num_output: 20
		kernel_size :5
		stride: 1
		weight_filer{        		//权重参数w的初始化方案，使用xavier算法
		 type: "xavier"
		}
		bias_filler{
		  type: "constant"			//偏执参数b初始化为常数，一般为0
		}
	}
}

#下采样层（pool）
layer{
	name: "pool1"
	type: "Pooling"
	bottom: "conv1"
	top："pool1"
	pooling_param{
		pool: MAX
		kernel_size: 2
		stride: 2
	}
}

#全连接层
layer{
	name: "ip1"
	type: "InnerProduct"
	bottom: "pool2"
	top："ip1"
	param{
		lr_mult: 1
	}
	param{
		lr_mult: 2
	}
	inner_product_param{
		num_output: 500
		weight_filer{
			type: "xavier"
		}
		bias_filler{
			type: "constant"
		}
	}
}

#非线性层
layer{
	name: "relu1"
	type: "ReLU"
	bottom: "ip1"
	top： "ip1"
}

#准确率层（计算准确率）
layer{
	name: "accuracy"
	type: "Accuracy"
	bottom: "ip2"
	bottom: "lable"
	top: "accuracy"
	include{
		phase: TEST
	}
}

#损失估计层
layer{
	name: "loss"
	type: "SoftmaxWithLoss"
	bottom: "ip2"
	bottom: "lable"
	top："loss"
}

#定义完网络模型，还需要配置关于模型优化的文件：
